src/plugins/graph/components/graph-axis.tsx

```diff
@@ -43,6 +43,7 @@ export const GraphAxis = observer(function GraphAxis({
     droppableId = `${instanceId}-${place}-axis-drop`,
     hintString = useDropHintString({role: axisPlaceToAttrRole[place]}),
     emptyPlotIsNumeric = useSettingFromStores("emptyPlotIsNumeric", "graph") as boolean | undefined,
+    attributeDnD = useSettingFromStores("attributeDnD", "graph") as boolean | undefined,
```

Comment:
If you reverse the polarity of the setting as I suggested above, then these all become:
```typescript
    enableAttributeDnD = !useSettingFromStores("disableAttributeDnD", "graph")
```
i.e. you eliminate the need for the casts.